modules/storage-persistent-storage-block-volume-examples.adoc

```diff
@@ -67,12 +69,13 @@ spec:
       persistentVolumeClaim:
         claimName: block-pvc <3>
 ----
-<1> `volumeDevices`, similar to `volumeMounts`, is used for block devices
-and can only be used with `PersistentVolumeClaim` sources.
-<2> `devicePath`, similar to `mountPath`, represents the path to the
-physical device.
+<1> `volumeDevices`, instead of `volumeMounts`, is used for block
```

Comment:
@huffmanca is the build failing because the callouts are before the text?

Reply:
It was failing due to an issue in the block volume table, where I was attempting to include the callout stanza after the table without defining them inside it.

This issue should be resolved.